v1/src/test/java/com/google/cloud/teleport/spanner/ExportPipelineIT.java

```diff
@@ -198,70 +201,23 @@ private void testSpannerToGCSAvroBase(
       Function<PipelineLauncher.LaunchConfig.Builder, PipelineLauncher.LaunchConfig.Builder>
           paramsAdder)
       throws IOException {
-    // Arrange
     String setDefaultTimeZoneStatement =
         "ALTER DATABASE db SET OPTIONS (default_time_zone = 'UTC')";
```

Comment:
nit: We should move this statement as well to `spanner-gsql-ddl.sql`

Reply:
UpdateDDLStatements call can fail in the middle, while processing many ddl statements and it can happen that some of the statements would have been applied. Now, when we get the RESOURCE_EXHAUSTED, we will re-apply all these ddl statements. And if then if this set default time zone statement is run on a non-empty db (containing some tables) this statement it will fail. Hence we have to apply it separately.